Adapt regression tests to new checkpoint format

emjotde opened the pull request:
This updates regressions tests to work with the new checkpoint format from the internal branch. 
@snukky the regression tests for exponential-smoothing require some more updates, since the model.npz.orig.npz file is now not saved separately but inside the checkpoint. I suppose we should check for the key?

emjotde committed:
adapt regression tests to new checkpoint format

emjotde committed:
Merge branch 'master' into mjd/fp16.2

snukky committed:
Merge branch 'master' into mjd/fp16.2

snukky committed:
Handle 'Synced seed' logs

snukky committed:
Fix tests affected by changed logging formats

snukky committed:
Merge branch 'master' into mjd/fp16.2

snukky reviewed (approved):
The tests work with the `mjd/fp16.4` branch.

snukky committed:
Update LICENSE.md

pedrodiascoelho committed:
add test for training with factors

pedrodiascoelho committed:
add test for decoding with factors

pedrodiascoelho committed:
remove debugging line

pedrodiascoelho committed:
reduce beam size for decoding test

pedrodiascoelho committed:
updated and simplified factors training test

- added clip-norm-0
- added verification if factors are being used
- factored vocabulary is no longer constructed on the fly. The one already in the models folder is used
- made training test marian-10 friendly

pedrodiascoelho committed:
add verification of factors activation on the decoder test

pedrodiascoelho committed:
All tests in a single folder now. Removed vocabs

snukky committed:
Remove two lines from factors/ tests

snukky committed:
Download new model 'factors'

snukky committed:
Merge branch 'mjd/fp16.2' into merge-fp16
